documentation/API-design-guidelines.md

```diff
@@ -1287,44 +1290,54 @@ response:
 
 
 ### 12.2 Event notification
 
 The event notification endpoint is used by the API server to notify the API consumer that an event occured.
 
-Note: The notification is the message posted on listener side. We describe the notification message in the CAMARA OAS but it could confusing because this endpoint should be implemented on the business API consumer side. This notice should be explicited mentioned in all CAMARA API documentation featuring notifications.
+CAMARA event notification leverages **[CloudEvents](https://cloudevents.io/)** as it is a vendor-neutral specification for defining the format of event data. A generic neutral CloudEvent notification swagger is available in commonalities/artifact directory (notification-as-cloud-event.yaml).
+
+Note: The notification is the message posted on listener side. We describe the notification(s) in the CAMARA API using the `callbacks`. From API consumer it could confusing because this endpoint must be implemented on the business API consumer side. This notice should be explicited mentioned in all CAMARA API documentation featuring notifications.
 
-Only Operation POST is provided for `eventNotifications` and the expected response code is `204`.
+Only Operation POST is provided for `eventNotifications` and the expected response code is `204`. 
+The url for this `POST` operation must be specified in the swagger as `{$request.body#/webhook/notificationUrl}`. 
 
-For consistence among CAMARA APIs the uniform `eventNotifications` model must be used:
+For consistence among CAMARA APIs the uniform CloudEvents model must be used with following rules:
 
 | name | type | attribute description | cardinality |
 | ----- |	-----  |	 -----  |  -----  | 
-| eventSubscriptionId | string | subscription identifier - could be valued for Resource-based subscription | optional |
-| event | object | event structure - see next table | mandatory |
+| id | string | identifier of this event, that must be unique in the source context. | mandatory |
+| source | string - URI | identifies the context in which an event happened in the specific Provider Implementation. Often this will include information such as the type of the event source, the organization publishing the event or the process that produced the event. The exact syntax and semantics behind the data encoded in the URI is defined by the event producer. | mandatory |
+| type | string | a value describing the type of event related to the originating occurrence. For consistency accross API we mandate following pattern: `org.camaraproject.<api-name>.<EVENT_NAME` (for exemple org.camaraproject.device-status.ROAMING_CHANGE_COUNTRY ) | mandatory |
+| specversion | string | version of the specification to which this event conforms (must be 1.0 if it conforms to cloudevents 1.0.2 version) | mandatory |
```

Comment:
Just as all Camara APIs conform to the same OAS version, we need to already fix the cloudevents spec version to the specific version number ( for e.g. 1.0.2) and not leave this up to the subprojects to choose on their own.

Reply:
It looks that CloudEvents specs are pretty stable - since 2019 they advanced from 1.0.0 to 1.0.2. Since the format is implemented in multiple systems they should keep the backward compatibility as much as possible. We can safely stay with specversion "1.0" even the new version is introduced since here we use only base functionality.

Reply:
Fixed the version to V1.0.
... and in improbable case of new version occurs for CloudEvents I guess we'll revisit the paragraph.